src/sysdiagnose/analysers/yarascan.py

```diff
@@ -3,8 +3,11 @@
 import glob
 import threading
 import queue
+import logging
```

Comment:
same here?

Reply:
that should be fixed with the commit [5b34d40](https://github.com/EC-DIGIT-CSIRC/sysdiagnose/pull/106/commits/5b34d40f75fbfd1a3d4c43dd8c03e90fc0064ca2)